common/models/data-source-definition.js

```diff
@@ -325,6 +325,7 @@ function ready(DataSourceDefinition) {
       }
 
       done.apply(self, msg.callbackArgs);
+      child.kill();
```

Comment:
yeah since the child isn't smart enough to figure it out when it should exit. which is the reason for this whole pr.